api/test/e2enew/route/route_with_management_fields_test.go

```diff
@@ -0,0 +1,458 @@
+		updatetime = gjson.Get(string(respBody), "data.update_time")
+		gomega.Expect(createtime.Int()).To(gomega.SatisfyAll(
+			gomega.BeNumerically(">=", time.Now().Unix()-1),
+			gomega.BeNumerically("<=", time.Now().Unix()+1),
+		))
+
+		gomega.Expect(updatetime.Int()).To(gomega.SatisfyAll(
+			gomega.BeNumerically(">=", time.Now().Unix()-1),
+			gomega.BeNumerically("<=", time.Now().Unix()+1),
+		))
+	})
+
+	table.DescribeTable("test for route with name description",
+		func(tc base.HttpTestCase) {
+			base.RunTestCase(tc)
+		},
+		table.Entry("update the route (r1)", base.HttpTestCase{
+			Object: base.ManagerApiExpect(),
+			Path:   "/apisix/admin/routes/r1",
+			Method: http.MethodPut,
+			Body: `{
+					"name": "route1",
+					"uri": "/hello",
+					"name": "new jack",
+					"desc": "new desc",
+					"upstream": {
+						"type": "roundrobin",
+						"nodes": {
+							"` + base.UpstreamIp + `:1980": 1
+						}
+					}
+				}`,
+			Headers:      map[string]string{"Authorization": base.GetToken()},
+			ExpectStatus: http.StatusOK,
+			Sleep:        time.Duration(2) * time.Second,
+		}),
+		table.Entry("access the route's uri (r1)", base.HttpTestCase{
+			Object:       base.APISIXExpect(),
+			Method:       http.MethodGet,
+			Path:         "/hello",
+			Headers:      map[string]string{"Authorization": base.GetToken()},
+			ExpectStatus: http.StatusOK,
+			ExpectBody:   "hello world",
+			Sleep:        base.SleepTime,
+		}),
+	)
+
+	ginkgo.It("get the updated route information", func() {
+		time.Sleep(time.Duration(100) * time.Millisecond)
+		basepath := base.ManagerAPIHost + "/apisix/admin/routes"
+
+		request, _ := http.NewRequest("GET", basepath+"/r1", nil)
+		request.Header.Add("Authorization", base.GetToken())
+		resp, _ := http.DefaultClient.Do(request)
+		respBody, _ := ioutil.ReadAll(resp.Body)
+		createtime2 := gjson.Get(string(respBody), "data.create_time")
+		updatetime2 := gjson.Get(string(respBody), "data.update_time")
+		//verify the route and compare result
+		gomega.Expect(gjson.Get(string(respBody), "data.name").String()).To(gomega.Equal("new jack"))
+		gomega.Expect(gjson.Get(string(respBody), "data.desc").String()).To(gomega.Equal("new desc"))
+		gomega.Expect(createtime2.String()).To(gomega.Equal(createtime.String()))
+		gomega.Expect(updatetime2.String()).NotTo(gomega.Equal(updatetime.String()))
+	})
+
+	ginkgo.It("delete the route (r1)", func() {
+		base.RunTestCase(base.HttpTestCase{
+			Object:       base.ManagerApiExpect(),
+			Method:       http.MethodDelete,
+			Path:         "/apisix/admin/routes/r1",
+			Headers:      map[string]string{"Authorization": base.GetToken()},
+			ExpectStatus: http.StatusOK,
+		})
+	})
+
+	ginkgo.It("verify delete route (r1) success", func() {
+		base.RunTestCase(base.HttpTestCase{
+			Object:       base.APISIXExpect(),
+			Method:       http.MethodGet,
+			Path:         "/hello",
+			Headers:      map[string]string{"Authorization": base.GetToken()},
+			ExpectStatus: http.StatusNotFound,
```

Comment:
better to check respond body for each case

Reply:
Done :)